### DISCUSSION TITLE
Getting older versions of saleor?

### DISCUSSION THREAD

**Discussion by gookenhaim** [saleor/saleor#6036] [General] [answered]
Hi! I remember last year's saleor was quite simple compared to what it is today. 
So I would like to download last year's versions and tweak and test. 

Where can I download them? Or is it prohibited to get older versions at all? Thanks

**Comment by IKarbowiak** [comment]
Hi, [here](https://github.com/mirumee/saleor/releases) you have all releases. So you can find the one from a year ago and download source code for it or if you have Saleor already cloned, you can just checkout to the specific release tag.

**Comment by karolkielecki** [comment, marked as answer]
@gookenhaim Hi there! You can also find the legacy version here: https://github.com/mirumee/legacy-views. Please bear in mind that we don't maintain and support the old version of Saleor. All new updates will be delivered only to Saleor 2.0+. I recommend you to consider using the latest stable version of Saleor.

**Reply by gookenhaim** [reply]
Thank you so much for your answer! Yeah I will keep your advice in mind!:)
